helm/README.md

```diff
@@ -0,0 +1,107 @@
+# Agent Check: helm
+
+## Overview
+
+This check monitors [helm][1] through the Datadog Agent.
+
+It mirrors the output values of [helm-exporter][8].  You should expect a metric value of -1
+if the helm chart fails to install.  Here is the complete status map:
+
+```py
+STATUS_MAP = {
+    "unknown": 0,
+    "deployed": 1,
+    "uninstalled": 2,
+    "superseded": 3,
+    "failed": -1,
+    "uninstalling": 5,
+    "pending-install": 6,
+    "pending-upgrade": 7,
+    "pending-rollback": 8,
+}
+```
+
+Because helm uses k8s secrets to store metadata, you'll also need to give the
+pod running the helm check permissions to list secrets.  That will involve
+* A cluster-role that can list secrets
```

Comment:
since this check must be run as a cluster check, we should recommend using dedicated [cluster check runners ](https://github.com/helm/charts/blob/master/stable/datadog/values.yaml#L840) with this extra permissions (list secrets)
the cluster check runners have a dedicated [service account ](https://github.com/helm/charts/blob/master/stable/datadog/templates/agent-clusterchecks-rbac.yaml) so users can/should only grant the secret permissions to the cluster check runners to be able to run this check.
Once this integration become available, we can add a config param to the [public helm chart of the datadog agent](https://github.com/helm/charts/tree/master/stable/datadog) to configure the cluster check runners with this extra permissions

Reply:
I agree.  I'm currently running it as a cluster check.